generated/libs.go

```diff
@@ -1,10 +0,0 @@
-package generated
```

Comment:
This should not be deleted, strange things happen if this file is removed in my experience